src/main/java/org/jabref/gui/slr/ManageStudyDefinition.fxml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ButtonType?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.DialogPane?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Tab?>
+<?import javafx.scene.control.TableColumn?>
+<?import javafx.scene.control.TableView?>
+<?import javafx.scene.control.TabPane?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.control.Tooltip?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import org.jabref.gui.icon.JabRefIconView?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.control.ScrollPane?>
+<DialogPane
+        xmlns="http://javafx.com/javafx/15.0.1"
+        xmlns:fx="http://javafx.com/fxml/1"
+        fx:controller="org.jabref.gui.slr.ManageStudyDefinitionView">
+    <content>
+        <BorderPane>
+            <center>
+                <TabPane
```

Comment:
I personally would prefer a more 'compact' style of the fxml file, like we have in other fxml. Makes it easier to read and understand.

Reply:
Is this current version now in compliance with that style? ^^

Reply:
Will work for now, but maybe sometime in the future we should see that we reach some common style in the resource files too. ;-)

Reply:
Yeah, we should add a common style to the formatting settings, as otherwise, autoformat will "stretch" it apart. (At least for my current settings setup, can you check this on your setup using Strg + Alt + L in IntelliJ? )

Reply:
Not sooner than tuesday. 😞

Reply:
Tried to compress the fxml a bit. At the end of the day, its probably always a matter of taste. I hope you dont mind.